.github/workflows/gcc.yml

```diff
@@ -40,7 +37,9 @@ jobs:
         ${CXX} --version
         scripts/set-up-python.sh --python=python3
```

Comment:
These two lines should no longer be needed here, and could be moved to the "Test" section.